doc/MANUAL.adoc

```diff
@@ -314,10 +314,47 @@ this:
    `$HOME/.ccache/ccache.conf`.
 5. Otherwise, if `XDG_CONFIG_HOME` is set then use
    `$XDG_CONFIG_HOME/ccache/ccache.conf`.
-6. Otherwise, use `%APPDATA%/ccache/ccache.conf` (Windows),
+6. Otherwise, use
    `$HOME/Library/Preferences/ccache/ccache.conf` (macOS) or
    `$HOME/.config/ccache/ccache.conf` (other systems).
 
+The location of the cache directory is determined as follows:
+
+1. `cache_dir` in the above found configuration if specified.
+2. `$CCACHE_DIR` if set.
+3. `$HOME/.ccache` if it exists.
+4. `$XDG_CACHE_HOME/ccache` if `$XDG_CACHE_HOME` is set.
+5. Otherwise `~/.cache/ccache` or `~/Library/Caches/ccache` on Apple systems.
+
+On Windows, this is the method used to find the configuration file:
+
+1. If `CCACHE_CONFIGPATH` is set, use that path.
+2. Otherwise, if the environment variable `CCACHE_DIR` is set then use
+   `%CCACHE_DIR%/ccache.conf`.
+3. Otherwise, if <<config_cache_dir,*cache_dir*>> is set in the secondary
+   (system-wide) configuration file then use `<cache_dir>\ccache.conf`. The
+   system-wide configuration on Windows is
+   `%ALLUSERSPROFILE%\ccache\ccache.conf` by default, the `ALLUSERSPROFILE`
+   environment variable is usually `C:\ProgramData`.
+4. Otherwise, if there is a legacy `%USERPROFILE%\.ccache` directory then use
+   `%USERPROFILE%\.ccache\ccache.conf`.
+6. Otherwise, if `%LOCALAPPDATA%\ccache\ccache.conf` exists it is used.
+7. Otherwise, if `%APPDATA%\ccache\ccache.conf` exists it is used.
+
+The cache directory on Windows is determined as follows:
+
+1. `cache_dir` in the above found configuration if specified.
+2. `%CCACHE_DIR%` if set.
+3. `%USERPROFILE%\.ccache` if it exists.
+4. Otherwise `%LOCALAPPDATA%\ccache`.
+
+*WARNING:* Previous builds of ccache for Windows defaulted to storing the cache
```

Comment:
The [AsciiDoc syntax for a warning admonition](https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#admonitions) is to use the plain string `WARNING:`.

```suggestion
WARNING: Previous builds of ccache for Windows defaulted to storing the cache
```